packages/@aws-cdk/aws-stepfunctions-tasks/test/eks/call.test.ts

```diff
@@ -0,0 +1,54 @@
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import * as cdk from '@aws-cdk/core';
+import { EksCall, MethodType } from '../../lib/eks/call';
+
+describe('Call an EKS endpoint', () => {
```

Comment:
missing tests for invalid service integration pattern (i.e. an error is thrown if `RUN_JOB` is selected)

Reply:
Will add tests for run a job pattern in next commit.